### DISCUSSION TITLE
Welcome to restaurant Discussions!

### DISCUSSION THREAD

**Discussion by olasunkanmi-SE** [olasunkanmi-SE/restaurant#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by agrahari-himanshu** [comment]
Hey @olasunkanmi-SE 

I'm Himanshu Agrahari, MEAN Full Stack developer in an organization from India

I recently started going through the code you have written to understand the restaurant model and it's business implementation approach to have some idea, and the approach you are using is really helpful to provide insight and gain insight on what i was looking for

I have some questions in mind which i would be asking in the discussion forum, looking forward for an active discussion :)

Thanks

**Reply by olasunkanmi-SE** [reply]
Hi @HimanshuAgrahari07 thanks for reaching out, I will be more than delighted to answer any questions you have on this project.
